Tests/SVT-HEVC_FunctionalTests.py

```diff
@@ -388,11 +404,11 @@ def check_seq_support(self, test_name, seq, enc_params):
             if encMode >= 12:
                 return -1
             if width*height < 3840*2160:
+                if encMode == 11:
+                    return -1
+            if width*height < 1920*1080:
                 if encMode == 10:
```

Comment:
encMode 11 is not allowed for  if width*height < 1920*1080:

Reply:
if width*height < 1920*1080 and encMode is set to 11, then it will caught in the check above:

if width*height< 3840*2160

Reply:
thank you:)